docs/application/dotnet/samples/tizen-samples.md

```diff
@@ -254,6 +252,7 @@
 			</td>
 			<td>
 			<p><strong>FirstScreen</strong></p>
+                        <p>FirstScreen is a sample application that demonstrates how to launcher the apps on the TV.</p>
```

Comment:
If FirstScreen sample app lets you to launch only one app on the TV, then please change the sentence to:

<p>FirstScreen is a sample application that demonstrates how to launch an app on the TV.</p>